bin/stencil-bundle

```diff
@@ -57,13 +43,17 @@ if (!themeConfig.configExists()) {
 
 configuration = themeConfig.getRawConfig();
 
+var bundle = new Bundle(themePath, themeConfig, configuration);
+
+
 if (configuration.css_compiler) {
-    tasks.css = getCssAssembleTask(configuration.css_compiler);
+    tasks.css = bundle.getCssAssembleTask(configuration.css_compiler);
 }
 
-tasks.templates = assembleTemplatesTask;
-tasks.lang = assembleLangTask;
-tasks.schema = assembleSchema;
+tasks.templates = bundle.assembleTemplatesTask.bind(bundle);
```

Comment:
Same here, could this all be put into the bundle class with a single exposed `bundleTheme`?